runtime/lua/vim/_defaults.lua

```diff
@@ -159,7 +159,16 @@ do
       vim.lsp.buf.rename()
     end, { desc = 'vim.lsp.buf.rename()' })
 
-    vim.keymap.set({ 'n', 'v' }, 'crr', function()
```

Comment:
was 'v' instead of 'x' intentional?

Reply:
No

Reply:
Besides, `<C-r>` in Select mode is a thing, so using `'x'` mode is probably safer.